### DISCUSSION TITLE
Cant judge if I’m doing ok (?)

### DISCUSSION THREAD

**Discussion by ebeng** [martomi/chiadog#217] [Q&A]
Hi all,

Harvesting on a few servers. 
1 server is a Full Node. 

Im getting the following messages daily from all the servers;

Hello farmer! 👋 Here's what happened in the last 24 hours:

Received ☘️: 0.00 XCH
Proofs 🧾: None
Search 🔍: 
	 - average: 0.39s over 9451 searches
	 - over 5s: 89 occasions (0.9%)
	 - over 15s: 28 occasions (0.3%)
Plots 🌱: 662, new: 10
Eligible plots 🥇: 1.26 average
Skipped SPs ✅️: None

Is this ok or not? Same message is on all the servers, as all of the servers are pointing to the same NFS servers with the plots. 

Also from time to time, Im also getting messsages like:

`Seeking plots took too long: 51.47434 seconds!`

```Experiencing networking issues? Harvester did not participate in any challenge for 87 seconds. It's now working again.```

And very limited these ones:

`Experiencing networking issues? Skipped 2+ signage points in the last hour.
`

Probably this is due to when writing the plots from harvester to NFS, any search on the NFS is limited due to bandwidth of the interface ports.

**Comment by alx741** [comment]
Yep, you're doing perfectly

**Reply by ebeng** [reply]
Thanks mate!
